corgi/collectors/brew.py

```diff
@@ -684,7 +684,10 @@ def get_component_data(self, build_id: int) -> dict:
             # If this is an "image" type build, it may be building a container image, ISO image,
             # or other types of images. Check the content generator name to determine where this
             # image was built, which indicates what type of image it is.
-            if build["cg_name"] != "atomic-reactor":
+            # Some builds such as 903565 have the cg_name field set to None
+            if build["cg_name"] is None or build["cg_name"] == "atomic-reactor":
```

Comment:
question: is an iso image a kind of component we want to represent someday ?

Reply:
I had a quick look at some of the iso images in brew, but it's wasn't immediately clear how we could ingest them based solely on the data in Brew, so I'm going to punt that to a future (yet to be specified) epic.